code/modules/mob/living/living_defines.dm

```diff
@@ -71,6 +71,8 @@
 	var/obj/screen/schematics_background
 	var/shown_schematics_background = 0
 
+	var/list/advanced_butchery = list() //Includes a list of the tools used to butcher the mob, detectable via autopsy scanner rather than examine.
```

Comment:
We really don't need a new list on every mob - I suggest having this start at null, and wrapping getting in a proc to only make the list as needed.